node.js/Typescript: '"@ai-sdk/provider"' has no exported member named 'LanguageModelV1Source'

### Description

Environment: node.js 18.x, VSCode, TypeScript

I installed:

```
npm i ai
npm i @ai-sdk/mistral
npm i zod
```

Versions:

```
"@ai-sdk/mistral": "^1.1.11",
"ai": "^4.1.41",

```

I import:

```
import {createMistral, MistralProvider} from "@ai-sdk/mistral";
import {embed} from "ai";

```

and get these compile errors:


```
node_modules/@ai-sdk/ui-utils/dist/index.d.ts:1:10 - error TS2724: '"@ai-sdk/provider"' has no exported member named 'LanguageModelV1Source'. Did you mean 'LanguageModelV1'?

1 import { LanguageModelV1Source, LanguageModelV1FinishReason, JSONValue as JSONValue$1 } from '@ai-sdk/provider';
           ~~~~~~~~~~~~~~~~~~~~~

node_modules/ai/dist/index.d.ts:5:204 - error TS2724: '"@ai-sdk/provider"' has no exported member named 'LanguageModelV1Source'. Did you mean 'LanguageModelV1'?

5 import { JSONValue, EmbeddingModelV1, EmbeddingModelV1Embedding, ImageModelV1, ImageModelV1CallWarning, LanguageModelV1, LanguageModelV1FinishReason, LanguageModelV1LogProbs, LanguageModelV1CallWarning, LanguageModelV1Source, LanguageModelV1ProviderMetadata, LanguageModelV1CallOptions, AISDKError, LanguageModelV1FunctionToolCall, JSONSchema7, ProviderV1, NoSuchModelError } from '@ai-sdk/provider';
```                                                                                                                  

And indeed `LanguageModelV1Source` is missing in `@ai-sdk/provider`, I cannot find the string in that library.




### Code example

import {createMistral, MistralProvider} from "@ai-sdk/mistral";
import {embed} from "ai";


### AI provider

@ai-sdk/mistral v1.1.11

### Additional context

Code example is a TypeScript file.

--- a/.changeset/soft-jokes-push.md
+++ b/.changeset/soft-jokes-push.md
@@ -1,0 +1,5 @@
+---
+'@ai-sdk/provider': patch
+---
+
+fix (ai/provider): publish with LanguageModelV1Source

